fixed typos in backup.rst file (#3636)
mattermost/docs: source/administration/backup.rst

Backup
------

> The state of your Mattermost server is contained in multiple data stores that need to be separately backed-up and restored to fully recover your system from failure. 

Reviewer: ```suggestion
The state of your Mattermost server is contained in multiple data stores that need to be separately backed up and restored to fully recover your system from failure. 
```

Automated backup
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Automating backup up a Mattermost server provides a copy of the server's state at a particular point in time, which can be restored if a failure in the future leads to loss of data. Options include: 

Reviewer: ```suggestion
Automating backup for a Mattermost server provides a copy of the server's state at a particular point in time, which can be restored if a failure in the future leads to loss of data. Options include: 
```

> - Automation to periodically backup the Mattermost server, which may include all the components listed above or a subset depending on what you choose to protect.

Reviewer: ```suggestion
- Automation to periodically back up the Mattermost server, which may include all the components listed above or a subset depending on what you choose to protect.
```
